language/examples/document-qa/question_answering_large_documents.ipynb

```diff
@@ -174,11 +174,11 @@
    },
```

Comment:
<p>Can you add for colab: 'or use the button to restart'</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/47/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
added.